src/backend/distributed/sql/citus--11.1-1--11.2-1.sql

```diff
@@ -11,3 +11,5 @@ DROP FUNCTION pg_catalog.worker_append_table_to_shard(text, text, text, integer)
 #include "udfs/citus_internal_adjust_local_clock_to_remote/11.2-1.sql"
 #include "udfs/worker_split_shard_replication_setup/11.2-1.sql"
 #include "udfs/citus_task_wait/11.2-1.sql"
+#include "udfs/citus_prepare_pg_upgrade/11.2-1.sql"
```

Comment:
For my edification, these two includes *must* always be the top lines in any new upgrade/downgrade scripts?

Reply:
Not necessarily. All changes happen transactionally.

If you rely on the new side effects during upgrading you'll need to add the directives before you use them.

Here we actually place them at the bottom, which ought to be working.

Reply:
I think my comment didn't convey it well, it's not the actual position in the file, but shouldn't these two lines *always* present in the downgrade/upgrade file(s). In other words, did the file creator missed it?

Reply:
The citus-11.1-1--11.2-1.sql file is/should be created right after a release. This is when any udf's are still the same old version, eg there was no `citus_prepare_pg_upgrade/11.2-1.sql` file. Instead an earlier version of Citus would install the latest version of the function that was available during the upgrade path.

Once we want to change a function we make the changes in their `latest.sql` file and copy this to a versioned filename for future use. When we have added [`citus_prepare_pg_upgrade/11.2-1.sql`](https://github.com/citusdata/citus/pull/6315/files#diff-7103e6c5be9129b5f9e011dc9cbe062fac7a4590e0db178e91d8c8fc5ade657e) (and some others) we failed to include those new versioned files here.

I am thinking of adding a stylechecker, or possibly add this to the udf sql style checker in the future, as to block a PR on missing includes.